common-arcana.lic

```diff
@@ -676,7 +641,7 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, tattoo_tm = false, data['runestone_name'], data['runestone_tm'])
```

Comment:
I also just noticed that you have `tattoo_tm = false`. It's probably safer to just use `false`.

Reply:
I think I copied it from line 112 because I accidentally deleted it here on one of the versions.

Reply:
Also made this `data['tattoo']`.